emscripten.py

```diff
@@ -661,6 +661,9 @@ def add_standard_wasm_imports(send_items_map):
       return value;
     }'''
 
+  if shared.Settings.INSTRUMENT:
+    send_items_map['log_function'] = '''log_function'''
```

Comment:
No need for triple quotes here.

Reply:
Where is `log_function` getting defined? Can we avoid needing to specify a special/dedicated mechanism to import this, but could instead use a regular JS library function for that?

I would be interested in overriding this `log_function` for my own debugging/tooling, and also perhaps the ones that `AUTODEBUG` provides, but it looks like overriding the functionality is not currently possible.

JS libraries have the convenience that if one specifies a `--js-library` file on the command line, it can override system JS library defined symbols, so one can override default behavior for the "I know what I'm doing" scenarios. That would be cool to support for this feature!